### PR TITLE
Add instructions on how to change default Karaf credentials

### PR DESCRIPTION
**Pull request by neykov**

### CODE REVIEW

File: guide/ops/production-installation.md [hunk 1]
```diff
@@ -81,6 +82,13 @@ It may be useful to use the following script to install an initial `brooklyn.pro
 % chmod 600 ~/.brooklyn/brooklyn.properties
 {% endhighlight %}
 
+### <a id="configuring-karaf-security"></a>Configuring Karaf Security
+
+Out of the box Brooklyn includes the default Karaf security configuration.
```
Comment (drigodwin) [inline guide/ops/production-installation.md]: Out of the box, Apache Brooklyn includes

File: guide/ops/production-installation.md [hunk 2]
```diff
@@ -81,6 +82,13 @@ It may be useful to use the following script to install an initial `brooklyn.pro
...
+It is used for connecting to the ssh port of Karaf (which is available
+to localhost connections only).
```
Comment (drigodwin) [inline guide/ops/production-installation.md]: How about:  This configuration is used manage connections to the ssh port of Karaf

File: guide/ops/production-installation.md [hunk 3]
```diff
@@ -81,6 +82,13 @@ It may be useful to use the following script to install an initial `brooklyn.pro
...
+It is recommended to update the credentials as detailed in the
```
Comment (drigodwin) [inline guide/ops/production-installation.md]: It is recommended that you update the

File: guide/ops/production-installation.md (hunk 1 above)
Comment (neykov) [inline guide/ops/production-installation.md]: Updated.

**Comment by neykov** [comment]
Thanks @drigodwin, updated.

**Pull request merged**